recipes/pygame/patches/0008-mark-some-custom-blitting-tests-as-broken.patch

```diff
@@ -0,0 +1,20 @@
+From 15f971121246b3bb59465c26e974eccc15a33226 Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Fri, 29 Jul 2022 10:45:27 +0200
+Subject: [PATCH 08/10] mark some custom blitting tests as broken
+
+---
+ test/surface_tags.py | 1 +
+ 1 file changed, 1 insertion(+)
+ create mode 100644 test/surface_tags.py
+
+diff --git a/test/surface_tags.py b/test/surface_tags.py
+new file mode 100644
+index 00000000..20d724a2
+--- /dev/null
++++ b/test/surface_tags.py
+@@ -0,0 +1 @@
++__tags__ = ["broken"]
+-- 
+2.37.0.windows.1
```

Comment:
This custom patch is not needed anymore, has an upstream fix (but you may have to use the latest pre-release as 2.1.2 doesn't have it)